config/ember-try.js

```diff
@@ -125,16 +11,26 @@ module.exports = {
         resolutions: {
           'ember': '~2.8.0'
         }
+      },
+      npm: {
+        devDependencies: {
+          'ember-source': null
+        }
       }
     },
     {
-      name: 'ember-2-9',
+      name: 'ember-lts-2.12',
       bower: {
         dependencies: {
-          'ember': '~2.9.0'
+          'ember': 'components/ember#lts-2-12'
         },
         resolutions: {
-          'ember': '~2.9.0'
+          'ember': 'lts-2-12'
```

Comment:
This can be updated to:

```
{
      name: 'ember-lts-2.12',
      npm: {
        devDependencies: {
          'ember-source': '~2.12.0'
        }
      }
    },
```